### ISSUE TITLE
InputRef for the dTV

### ISSUE THREAD

**Issue by epapoutsellis** [vais-ral/CCPi-Regularisation-Toolkit#157]
I think, we should raise an error if the `InputRef` does not have the same size as `Input` in [here](https://github.com/vais-ral/CCPi-Regularisation-Toolkit/blob/master/src/Core/regularisers_CPU/FGP_dTV_core.c#L46).

**Comment by paskino** [comment]
Maybe this could be handled easily at the [Cython ](https://github.com/vais-ral/CCPi-Regularisation-Toolkit/blob/master/src/Python/src/cpu_regularisers.pyx#L98)or Python level.

**Comment by dkazanc** [comment]
makes sense to do it in Cython wrapper, yes. It's on the list of things together with anatomical priors.
